### ISSUE TITLE
Xcode 11 warning for Optional.none in generated code

### ISSUE THREAD

**Issue by cltnschlosser** [apollographql/apollo-ios#724]
`Assuming you mean 'Optional<Optional<MyType>>.none'; did you mean 'MyType.none' instead?`

```
public var myProperty: Swift.Optional<MyType?> {
    get {
      return graphQLMap["myProperty"] as? Swift.Optional<MyType?> ?? .none
    }
    set {
      graphQLMap.updateValue(newValue, forKey: "myProperty")
    }
  }
```

The codegen should generate `Optional<MyType?>.none` (or maybe `?? nil`) instead of `.none`.

**Comment by designatednerd** [comment]
Yeah when I updated this it was originally returning `nil`, but `nil` is not actually a valid return type for `Swift.Optional`, at least in Swift 5. I'm not sure what's changed in 5.1 to elicit this warning.

I'll take a look at this when I get back!

**Comment by cltnschlosser** [comment]
The warning only shows up when MyType is an enum that has a none case. See https://bugs.swift.org/browse/SR-2176
